internal/scm/github/github.go

```diff
@@ -187,9 +197,36 @@ func (g *Github) GetRepositories(ctx context.Context) ([]scm.Repository, error)
 	return repos, nil
 }
 
+func (g *Github) excludeRepositoryFilter(repoName string) (bool, error) {
```

Comment:
There is no reason it exclusively do this for GitHub when it's not SCM specific. The filtering should be done in the multi-gitter package.

Reply:
Had a feeling it could be used elsewhere but since my focus was on enhancing Github Search API I just pushed it off. Thanks for the feedback!, will do the recommend changes 🙂